lib/farscape/transition.rb

```diff
@@ -23,8 +23,12 @@ def invoke(args={})
       call_options[:url] = @transition.uri
       call_options[:method] = @transition.interface_method
       call_options[:headers] = @agent.get_accept_header(@agent.media_type).merge(options.headers || {})
-      call_options[:params] = options.parameters unless options.parameters.blank?
       call_options[:body] = options.attributes unless options.attributes.blank?
+      if @transition.interface_method.to_s.downcase == 'get'
+        call_options[:params] = options.parameters unless options.parameters.blank?
+      else
+        call_options[:body] = (call_options[:body] || {}).merge(options.parameters) unless options.parameters.blank?
```

Comment:
I'm not clear on why we need this line. Why can't you just pass the body in using attributes and leave params empty on a POST?